list_updatable_packages

```diff
@@ -100,9 +136,40 @@ def build_matrix(specs: Iterable[Spec]) -> Generator[dict, None, None]:
                 }
                 yield entry
 
+def build_dependency_matrix(gems: dict, project: str='foreman') -> Generator[dict, None, None]:
+    if DEBUG: print(f'Building dependency matrix for {project}')
+    for gem in gems:
+        try:
+            name = gem['name']
+            path = Path(f'packages/{project}/rubygem-{name}/rubygem-{name}.spec')
```

Comment:
I'm not a big fan of doing the path construction using strings. A more native way would be:
```suggestion
            path = Path('packages') / project / f'rubygem-{name}' / f'rubygem-{name}.spec'
```